conans/test/utils/tools.py

```diff
@@ -706,6 +706,11 @@ def created_package_revision(self, ref):
                                str(self.out)).group(1)
         return package_id
 
+    def created_package_reference(self, ref):
```

Comment:
You can call this in the `TestClient` object to obtain the `PkgReference` just created.